### ISSUE TITLE
String Read Time

### ISSUE THREAD

**Issue by Sherlouk** [BasThomas/Analysis#19]
Websites such as Medium estimate a read time based on the number of words and an estimated words per minute (~200). I think it could be quite a nice feature to return the amount of seconds it would take to read the string taking into account the reading ease of the string

**Comment by BasThomas** [comment]
Yeah sounds cool, especially if there’s some standard or system to calculate this!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
